Feat/LDG-723-add-berachain-to-ledger-asset-dapps

keiff3r opened the pull request.

n4l5u0r reviewed (commented):
@GuilaneDen pourquoi tu enleve swapOnUniswapV2Fork, swapOnUniswap etc…?

GuilaneDen commented:
Une demande de Ledger parce que c'est fonction ne sont pas implémenté sur Base. Mais l'explorer Base est bloqué sur ma machine donc je pouvais pas vérifier.

reffiek committed:
feat: add support for Berachain's contract

The four contracts added are:
-BGT token
-Honey Factory (ERC1967Proxy)
-Reward Vault
-Reward Vault Factory  (ERC1967Proxy)

reffiek committed:
feat(berachain): add abis for BGT, Honey Factory, RewardVault and Reward Vault Factory

reffiek committed:
Merge remote-tracking branch 'origin/main' into feat/LDG-723-add-berachain-to-ledger-asset-dapps
